src/video_stream.cpp

```diff
@@ -291,11 +291,20 @@ virtual void subscribe() {
     cap->set(cv::CAP_PROP_FRAME_HEIGHT, latest_config.height);
   }
 
+  // Normalize property ranges (see https://github.com/ros-drivers/video_stream_opencv/pull/102/files)
+  cap->set(cv::CAP_PROP_MODE, true);
```

Comment:
is seems like this became necessary at ROS Noetic, see the link above for better understanding

Reply:
The issue is that opencv would send normalized parameters to the camera (from 0.0 to 1.0) and it would be translated for the camera's range (for instance, it could be from 0 to 1000). However, since Noetic this doesn't happen anymore, and `video_stream_opencv` actually tries to set the parameters to 0.0 or 1.0